indexer/services/comlink/__tests__/controllers/api/v4/transfers-controller.test.ts

```diff
@@ -824,6 +834,8 @@ describe('transfers-controller#V4', () => {
       await WalletTable.create({
         address: testConstants.defaultWalletAddress,
         totalTradingRewards: '0',
+        totalVolume: '0',
+        isWhitelistAffiliate: false,
```

Comment:
nit: these test object are getting big, can you either use defaultWallet or create a constant in the test that is referenced. Please apply across all test files